src/readers/src/main/python/PowerBIWriter.py

```diff
@@ -0,0 +1,28 @@
+# Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
Put it into a submodule so it doesn't looks like:
``py
import mmlspark;
mmlspark.PowerBIWriter(...)
``

Reply:
Again this should look like mmlspark.PowerBIWriter.writeToPowerBI(....)
or df.writeToPowerBI(....)